### ISSUE TITLE
using mobilenetv3 + yolov5 

### ISSUE THREAD

**Issue by lana211191** [ultralytics/yolov5#6847]
_Disclaimer: I am a bit of a novice , i started learning deep learning 7 month ago so i might have some knowledge gaps_

Hello, i want to use Mobilenetv3 with yolov5, i adjusted the code but i have an issue.
I understand that now that i changed theh backbone to mobilenet i cannot use the yolov5 pretrained weights since there is no common/similar type of  layers in the backbone.

i was wondering if i can use mobilenet v3 pretrained weights (downloaded from pytorch) and freeze a part of the layers ?

If yes how can i transform the .pth file of mobilenet to .pt (accepted by yolov5)
(**error that i get**: mobilenet_v3_large-8738ca79.pth acceptable suffix is ['.pt'])
Thank you

**Comment by glenn-jocher** [comment]
@lana211191 YOLOv5 doesn't recognize *.pth suffixes which is causing your error. You can simply rename your file to end with *.pt to avoid this. 

Attempting to transfer weights is a different story, you'd have to implement this feature yourself in train.py here:
https://github.com/ultralytics/yolov5/blob/601dbb83f01b58355211f2565cfa4eecb48b1220/train.py#L117-L132

**Comment by zhiqwang** [comment]
FYI @lana211191 and @glenn-jocher ,

We propose a new way to construct a YOLOv5 Lite model with TorchVision's pre-trained MobileNetV3-Large FPN backbone. Concretely we restructured the  YOLOv5's model into following four sub-modules in the layout of TorchVision, so it could better load the pre-trained models published by TorchVision. Maybe this can also used as a plugin for YOLOv5. See https://github.com/zhiqwang/yolov5-rt-stack/pull/342 and https://github.com/zhiqwang/yolov5-rt-stack/pull/343 for more details.

- [BackboneWithPAN](https://github.com/zhiqwang/yolov5-rt-stack/blob/de119660874fdf51c0c5b0ff7fa308597be6d636/yolort/models/backbone_utils.py#L11-L32)
- [YOLOHead](https://github.com/zhiqwang/yolov5-rt-stack/blob/de119660874fdf51c0c5b0ff7fa308597be6d636/yolort/models/box_head.py#L13-L20)
- [PostProcess](https://github.com/zhiqwang/yolov5-rt-stack/blob/de119660874fdf51c0c5b0ff7fa308597be6d636/yolort/models/box_head.py#L356-L359)
- [SetCriterion](https://github.com/zhiqwang/yolov5-rt-stack/blob/de119660874fdf51c0c5b0ff7fa308597be6d636/yolort/models/box_head.py#L81-L97)

The `SetCriterion` is almost same with YOLOv5's [`ComputeLoss`](https://github.com/ultralytics/yolov5/blob/8a66ebad44e8ecf90c7d27757c832579398d4baf/utils/loss.py#L91-L92), the only differences is the output type.
